docs/assisted-deployment.md

```diff
@@ -0,0 +1,44 @@
+Assisted Deployment
+==
+
+The Assisted Deployment refers to the use of [assisted
+service](https://github.com/openshift/assisted-service/) for deploying OpenShift nodes. Please, read
+the Assisted Service documentation for more information about its architecture and how it works.
+
+Pre-requisites
+==
+
+2 disks need to be created and added to the worker nodes. This can be done either using
+`virt-manager` or libvirt directly. The disks should be of, at least, 10GB each and they should be
+attached to the `/dev/sdb` and `/dev/sdc` devices respectively. Please update
+`assisted_deployment.sh` if this is not the case.
+
+Note that there is work in progress to automate the disk creations and the device name
```

Comment:
Might be appropriate to create an issue and link it here.

Reply:
done :)